config/manager/manager.yaml

```diff
@@ -29,6 +29,7 @@ spec:
         args:
         - --leader-elect
         - --zap-log-level=$(ZAP_LOG_LEVEL)
+        - --MaxConcurrentReconciles=20
```

Comment:
+1 to what Giulio mentioned, the env var could be defined like $(ZAP_LOG_LEVEL), and the value could be initialized in the `params.env`. You will also need to add a reference in the base kustomization file.